docs/network/wwivnet_linux.md

```diff
@@ -103,105 +51,112 @@ dosemu -dumb -quiet "$(basename ${0}).bat ${NET}" 2>/dev/null
+* You will be prompted to pick the net (WWVIVnet) and confirm the system (Weather Station).
+* Give it a subject. Send the Net Coordinator a love note and hit /S to save and send.
+* If you and 1@1 are chatting in IRC, you can confirm and troubleshoot all at once. 
+Otherwise, please give 1@1 at least several hours to read and reply.  
 
-program arguments:
---allow_sendback       Allow sendback (only used by legacy network0)
---bbsdir               (optional) BBS directory if other than current directory
---callout_time         Start time of the callout (only used by legacy network0)
---help                 displays help.
---network              Network name to use (i.e. wwivnet).
---network_number       Network number to use (i.e. 0).
---node                 Network node number to dial.
---phone_number         Network number to use (only used by legacy network0)
---speed                Modem Speedto use (only used by legacy network0)
-/N####     Network node number to dial.
-.####      Network number (as defined in wwivconfig)
-```
+## Mail Processing
 
+Network mail processing **must** be done from ${WWIV_DIR}.  The basic command 
+to invoke message transport is:
 
-Full networkb (the command that network calls to do the work) usage looks like:
+`network --net=(name or position of your network) --node=(number of target)`
 
-```
-Usage: networkb [flags]
-Flags:
---network  Network name to use (i.e. wwivnet)
---bbsdir   (optional) BBS directory if other than current directory 
---send     Send network traffic to --node
---receive  Receive from any node
---node     Node number (only used when sending)
---port     Port number to use (receiving only)
---skip_net Skip invoking network1/network2/network3
-```
+So to continue with the example above, it could be:
 
-If you want to allow your system to listen for incoming connections to BinkP, that's possible to do with the --receive flag; but it's a one-shot service, so it needs to be put in a loop (more details to come on this).  Most people will be perfectly fine with just running the network command on a schedule.
+`network --net=wwivnet --node=1`
 
-_**Putting it all together**_
+OR
 
-How often you schedule mail checks is entirely up to you, but every 15 minutes or so is probably more than sufficient.  Just adding the network command to cron should work nicely.
+`network --net=0 --node=1`
 
-At this point, WWIVnet should be working well enough to handle sending mail between systems. If you are still having issues, you will need to resolve them before moving on to Message Subs. Look for help in the irc channel if you are stuck.
+You should call this at a regular but decently spread out interval. (Every quarter 
+hour is *more* than sufficient.)  You can use [cron](https://linuxconfig.org/using-cron-scheduler-on-linux-systems), 
+[systemd](https://linuxconfig.org/how-to-schedule-tasks-with-systemd-timers-in-linux), or 
+another process to call this command from ${WWIV_DIR}.
```

Comment:
If you use wwivd, you can also let it manage callouts too (it'll use the config specified in callout.net)

Reply:
Oh, I see it now!  Yeah, let me add that in here.... I am presuming just changing net-callouts to "yes" is enough?